android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIScrollView.java

```diff
@@ -1121,4 +1122,13 @@ public void registerForKeyPress()
 			registerForKeyPress(this.scrollView);
 		}
 	}
+
+	@Override
+	public KrollDict toImage() {
+		if (scrollView instanceof TiVerticalScrollView) {
+			return TiUIHelper.viewToImage(proxy.getProperties(), ((TiVerticalScrollView) scrollView).getLayout());
```

Comment:
Why not introduce a very simple interface internally here that lists the methods common between the two implementations:
```java
public interface TiScrollView {
  public TiScrollViewLayout getLayout();
}
```

It looks like this code isn't the only place where we're doing `instanceof` checks just to cast and ask for the exact same method signature. I imagine you could clean up the code nicely by introducing it and having the `scrollView` field be declared as a `TiScrollView`.